test/e2e/test-bento.js

```diff
@@ -10,22 +10,11 @@ describes.endtoend(
       // is attached as soon as possible.
       // (Feel free to increase if flaky.)
       this.timeout(100);
-
-      const shadowRoot = await env.controller.evaluate(
-        () =>
-          new Promise((resolve) => {
-            function getShadowRoot() {
-              const {shadowRoot} = document.body.firstElementChild;
-              if (shadowRoot) {
-                resolve(shadowRoot);
-              } else {
-                setTimeout(getShadowRoot, 15);
-              }
-            }
-            getShadowRoot();
-          })
+      await env.controller.switchToShadowRoot(
+        await env.controller.findElement('bento-timeago')
       );
-      await expect(shadowRoot).ok;
+
+      await expect(await env.controller.findElement('time')).ok;
```

Comment:
Does this work instead?
```suggestion
      await expect(await env.controller.findElement(':first-child')).ok;
```